### ISSUE TITLE
Accessibility: View title bar is not accessible with Orca

### ISSUE THREAD

**Issue by parkdoje** [parkdoje/Labeling_bot#31]
OS: Ubuntu 18.04
Screen Reader: Orca
Bug:
Open File Explorer
Tab through the Views, Open Editors, Explorer, Outline, Timeline
Orca doesn't read the name for each view title, while Voice Over on macOS does.
